data/json/recipes/recipe_electronics.json

```diff
@@ -1416,6 +1416,35 @@
     [ [ "amplifier", 1 ] ],
     [ [ "cable", 5 ] ]
   ]
+},{
+  "type": "recipe",
+  "result": "motor_micro",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_COMPONENTS",
+  "skill_used": "electronics",
+  "skills_required": [
+    "fabrication",
+    1
```

Comment:
I get that the file isn't linted, but there's no need to put the skill requirement on a separate line.

Reply:
Understood. That's what the linter made, didn't want to touch in case any keystroke would cause whatever issue that'd require linting anew.

Reply:
I am using linter made by @ZhilkinSerg on the basis of the stuff that's normally used and also get the same result on Cataclysm: Dark Days Ahead JSON Web Linting Tool. Possible OS (especially since I am not using English-languaged version) weirdness?